src/lib/main.js

```diff
@@ -101,22 +101,23 @@ import * as stateMappings from './stateMappings/index.js';
  *   }
  * });
  * ```
- * We forward `algoliasearch` which is the original algoliasearch module imported inside instantsearch.js
+ * We forward `algoliasearch`, which is the original Algolia search module imported inside InstantSearch.js
```

Comment:
was doubting about this change, but I think it's good

Reply:
algoliasearch refers directly to the npm module. Maybe this could be improved like so: 

```md
which is the original [Algolia search client](https://www.algolia.com/doc/api-client/javascript/getting-started/) imported inside InstantSearch.js
```